### PR TITLE
electrum: trezor wallet support & optional UIs

### PR DESCRIPTION
**Pull request by joachifm**
Depends on https://github.com/NixOS/nixpkgs/pull/7094. @ehmry

### CODE REVIEW

**Comment by ehmry** [comment]
It builds and works for me, I think it would be better to have PyQt4 on by default. The Trezor plugin needs it for it's pin entry dialog so I made it implicit with that.

**Comment by joachifm** [comment]
Okay, I've pushed an amended version which enables qt4 by default.

**Comment by ehmry** [comment]
If you cherry pick https://github.com/ehmry/nixpkgs/commit/50b081de819ff0cbff909d78ae74746a7fe4fca9 I'll close my PR

**Comment by joachifm** [comment]
On Wed, Apr 1, 2015, at 12:20 AM, Emery Hemingway wrote:
> If you cherry pick
> https://github.com/ehmry/nixpkgs/commit/50b081de819ff0cbff909d78ae74746a7fe4fca9
> I'll close my PR

Done

**Comment by ehmry** [comment]
:+1:

**Comment by joachifm** [comment]
I have removed the use of config, as that's frowned upon ...

**Comment by joachifm** [comment]
Rebased to resolve conflicts. Note that the travis build fails due to OOM.

**Comment by nbp** [comment]
@ehmir can you verify this pull request locally, and ask additional reviewers if needed for merging?

**Comment by joachifm** [comment]
I have pushed a rebased version once again. I have tested both the headless and QT4 UI.
cc @ehmry

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -7542,6 +7542,30 @@ let
   };
 
 
+  hidapi =
+  let rev = "78722d3c6c028cd66e3ecc1ee9fa03e63e671de9"; in
+  buildPythonPackage rec {
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: you can just put the rev straight away in fetchgit, no need to inherit.

**Comment by FRidh** [comment]
you might want to clean up indentation and where you put the brackets. Furthermore, check the nixpkgs manual for the guidelines regarding commit naming.

**Comment by joachifm** [comment]
Closing this, as most of it is now obsolete.

**Pull request closed without merging**